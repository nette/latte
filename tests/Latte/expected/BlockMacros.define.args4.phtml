<?php
%A%

final class Template%a% extends Latte\Runtime\Template
{

	public function main(): array
	{
		extract($this->params);
		echo '
named arguments import

';
		$this->createTemplate("import.latte", $this->params, "import")->render();
		echo '
a) ';
		$this->renderBlock($__nm = 'test', [1, 'var1' => 2] + [], 'html');
		echo '

b) ';
		$this->renderBlock($__nm = 'test', ['var2' => 1] + [], 'html');
		echo '

c) ';
		$this->renderBlock($__nm = 'test', ['hello' => 1] + [], 'html');
		return get_defined_vars();
	}

}
