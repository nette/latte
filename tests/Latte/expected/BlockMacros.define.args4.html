
named arguments import


a) 	Variables 1, ,


b) 	Variables , 1,


c) 	Variables , ,
